src/_igraph/graphobject.c

```diff
@@ -13728,6 +13799,22 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "@return: C{True} if it is complete, C{False} otherwise.\n"
    "@rtype: boolean"},
 
+  {"is_clique", (PyCFunction) igraphmodule_Graph_is_clique,
+   METH_VARARGS | METH_KEYWORDS,
+   "is_clique(vertices, directed=False)\n--\n\n"
```

Comment:
`is_clique()` supports `vertices=None` and `vertices` itself is optional. Here it seems that `vertices` is mandatory. I don't mind either way, you can make it mandatory or you can leave it as-is (when it supports `None`), just make sure that the docs are consistent with the behaviour.

Reply:
Fixed.